circle.yml

```diff
@@ -21,10 +21,12 @@ deployment:
     owner: drupalmel
     commands:
       - chmod +x ~/$CIRCLE_PROJECT_REPONAME/tests/deploy.sh
-      - BEET_VERSION="0.1.$CIRCLE_BUILD_NUM" BEET_BOX=beetbox-dev ~/$CIRCLE_PROJECT_REPONAME/tests/deploy.sh
+      - BEET_VERSION="0.1.$CIRCLE_BUILD_NUM" BEET_BOX=beetbox-dev BEET_TEMPLATE=teplate.json ~/$CIRCLE_PROJECT_REPONAME/tests/deploy.sh
```

Comment:
Thanks! -- we probably need a test for this ;)

On 2 February 2016 at 06:56, Oskar Schöldström <notifications@github.com>
wrote:

> In circle.yml
> <https://github.com/drupalmel/beetbox/pull/90#discussion_r51470992>:
>
> > @@ -21,10 +21,12 @@ deployment:
> >      owner: drupalmel
> >      commands:
> >        - chmod +x ~/$CIRCLE_PROJECT_REPONAME/tests/deploy.sh
> > -      - BEET_VERSION="0.1.$CIRCLE_BUILD_NUM" BEET_BOX=beetbox-dev ~/$CIRCLE_PROJECT_REPONAME/tests/deploy.sh
> > +      - BEET_VERSION="0.1.$CIRCLE_BUILD_NUM" BEET_BOX=beetbox-dev BEET_TEMPLATE=teplate.json ~/$CIRCLE_PROJECT_REPONAME/tests/deploy.sh
>
> Typo
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/drupalmel/beetbox/pull/90/files#r51470992>.
>



-- 


Thom Toogood
about.me/thomtoogood
[image: Thom Toogood on about.me]
  <http://about.me/thomtoogood>